src/gui/static/src/app/components/pages/wallets/wallet-detail/wallet-detail.component.ts

```diff
@@ -1,26 +1,30 @@
 import { Component, Input, OnDestroy } from '@angular/core';
-import { MatDialog, MatDialogConfig } from '@angular/material/dialog';
-import { ChangeNameComponent, ChangeNameData } from '../change-name/change-name.component';
-import { PasswordDialogComponent, PasswordDialogParams } from '../../../layout/password-dialog/password-dialog.component';
-import { NumberOfAddressesComponent } from '../number-of-addresses/number-of-addresses';
+import { MatDialog } from '@angular/material/dialog';
 import { TranslateService } from '@ngx-translate/core';
-import { HwWalletService } from '../../../../services/hw-wallet.service';
 import { Observable, SubscriptionLike } from 'rxjs';
+import { Router } from '@angular/router';
+import { mergeMap, first } from 'rxjs/operators';
+
+import { ChangeNameComponent, ChangeNameData } from '../change-name/change-name.component';
+import { PasswordDialogComponent, PasswordDialogParams, PasswordSubmitEvent } from '../../../layout/password-dialog/password-dialog.component';
+import { NumberOfAddressesComponent, NumberOfAddressesEventData } from '../number-of-addresses/number-of-addresses';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skycoin/pullRequest?prid=7163600)
```suggestion
import { NumberOfAddressesComponent, NumberOfAddressesEventData } from "../number-of-addresses/number-of-addresses";
```